recipes/trogon/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "trogon" %}
+{% set version = "636f114655da84c82f3d31649ee7165605349558" %}
```

Comment:
```suggestion
```

Reply:
This line is not used here at all BTW. Also, should do https://github.com/conda-forge/staged-recipes/pull/23391#discussion_r1305545629